include/cutlass/detail/helper_macros.hpp

```diff
@@ -62,7 +62,7 @@
 
 #if defined(CUTLASS_ENABLE_SYCL)
 #define CUTLASS_HOST
-#define CUTLASS_GLOBAL
+#define CUTLASS_GLOBAL __attribute__((always_inline)) inline
```

Comment:
That change is not needed the launch function has a Lamba which inside that lambda it has been annotated with 
``` [[clang::always_inline]] F(args...);``` so you dont need this attribute here